lib/_emerge/main.py

```diff
@@ -51,6 +51,7 @@
     "--skipfirst",
     "--tree",
     "--unordered-display",
+    "--usepkg-exclude-live",
```

Comment:
Do you anticipate that a user would want to set this in EMERGE_DEFAULT_OPTS and override it via the command line? If so, then it would be useful to make this a `--usepkg-exclude-live=<y|n>` option.